web/app/views/environments/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="card" >
+    <div class="card-block" >
+	<h4 class="card-title">Create a new Environment for <%= @app.name %></h4>
+	<%= form_for :environment, url: application_environments_path do |f| %>
+	    <% if @form.errors.any? %>
+		<div class="alert alert-danger" role="alert" >
+		    <strong>Whoops!</strong> <%= pluralize(@form.errors.count, "error") %> prevented
+		    this environment from being created.
+		</div>
+	    <% end %>
+	    <div class="form-group required<%= ' has-danger' if @form.errors.include?(:name) %>">
+		<%= f.label :name, class: "col-form-label" %>
+		<%= f.text_field :name, value: @form.name, class: "form-control form-control-lg" %>
+		<% @form.errors.full_messages_for(:name).each do |msg| %>
+		    <div class="form-control-feedback"><%= msg %></div>
+		<% end %>
+	    </div>
+	    <div class="form-group <%= ' has-danger' if @form.errors.include?(:productionFlag) %>">
+		<%= f.label "Production", class: "col-form-label" %>
+		<%= f.check_box :productionFlag, value: @form.productionFlag, class: "form-control" %>
+		<% @form.errors.full_messages_for(:productionFlag).each do |msg| %>
+		    <div class="form-control-feedback"><%= msg %></div>
+		<% end %>
+	    </div>
```

Comment:
We removed the production flag field in another PR, so let's remove this form group from here.